src/Core/AdminConsole/Services/Implementations/RabbitMqEventHttpPostListener.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Net.Http.Json;
+using Bit.Core.Models.Data;
+using Bit.Core.Settings;
+using Microsoft.Extensions.Logging;
+
+namespace Bit.Core.Services;
+
+public class RabbitMqEventHttpPostListener : RabbitMqEventListenerBase
+{
+    private readonly IHttpClientFactory _httpClientFactory;
+    private readonly string _httpPostUrl;
+    private readonly string _queueName;
+
+    protected override string QueueName => _queueName;
+
+    public const string HttpClientName = "EventHttpPostListenerHttpClient";
+
+    public RabbitMqEventHttpPostListener(
+        IHttpClientFactory httpClientFactory,
+        ILogger<RabbitMqEventListenerBase> logger,
+        GlobalSettings globalSettings)
+        : base(logger, globalSettings)
+    {
+        _httpClientFactory = httpClientFactory;
+        _httpPostUrl = globalSettings.EventLogging.RabbitMq.HttpPostUrl;
+        _queueName = globalSettings.EventLogging.RabbitMq.HttpPostQueueName;
+    }
+
+    protected override async Task HandleMessageAsync(EventMessage eventMessage)
+    {
+        using var httpClient = _httpClientFactory.CreateClient(HttpClientName);
+        var content = JsonContent.Create(eventMessage);
+        var response = await httpClient.PostAsync(_httpPostUrl, content);
+        response.EnsureSuccessStatusCode();
+    }
```

Comment:
We can just get the HttpClient a single time in the constructor like so:

```suggestion
    private readonly HttpClient _httpClient;
    private readonly string _httpPostUrl;
    private readonly string _queueName;

    protected override string QueueName => _queueName;

    public const string HttpClientName = "EventHttpPostListenerHttpClient";

    public RabbitMqEventHttpPostListener(
        IHttpClientFactory httpClientFactory,
        ILogger<RabbitMqEventListenerBase> logger,
        GlobalSettings globalSettings)
        : base(logger, globalSettings)
    {
        _httpClient = httpClientFactory.CreateClient(HttpClientName);
        _httpPostUrl = globalSettings.EventLogging.RabbitMq.HttpPostUrl;
        _queueName = globalSettings.EventLogging.RabbitMq.HttpPostQueueName;
    }

    protected override async Task HandleMessageAsync(EventMessage eventMessage)
    {
        var content = JsonContent.Create(eventMessage);
        var response = await _httpClient.PostAsync(_httpPostUrl, content);
        response.EnsureSuccessStatusCode();
    }
```